adcc/OperatorIntegrals.py

```diff
@@ -96,11 +96,24 @@ def available(self):
         Which integrals are available in the underlying backend
         """
         ret = []
-        for integral in ["electric_dipole"]:
+        for integral in ["electric_dipole", "magnetic_dipole", "momentum"]:
             if hasattr(self.provider_ao, integral):
                 ret.append(integral)
         return ret
 
+    def import_dipole_like_operator(self, ao_operator, is_symmetric=True):
```

Comment:
It cannot go in there because it only applies to operators with three Cartesian components. Say we'd like to import some other operator (e.g., from PE or PCM), we'd still need `replicate_ao_block` and only have one component.

Reply:
Agreed! Let's leave it as is. Maybe make it a free function though as it does not really depend on the self?

Reply:
It depends on `self.mospaces`, `self.__coefficients`, and `self.__conv_tol`.

Reply:
Ok that just leave it.